src/main/kotlin/org/eclipse/uprotocol/transport/validator/UAttributesValidator.kt

```diff
@@ -162,14 +148,33 @@ sealed class UAttributesValidator {
         }
```

Comment:
Actually the lowest priority is CS1 and not CS0 per the spec here: https://github.com/eclipse-uprotocol/up-spec/blob/main/basics/qos.adoc#default-priority